config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
+      autoInject: true
```

Comment:
It was `view` that caused the problem. My thought was that it might be easier for newer folks to the ecosystem if we could "construct" the initializer for them, but I'm also totally fine with providing docs on how to setup an initializer should you need one.